examples/android/app/src/main/jni/main.cpp

```diff
@@ -153,6 +153,8 @@ int main(int argc, char* argv[])
 #endif
                     }
                     break;
+                default:
+                    break;
```

Comment:
Why did you add this?

Reply:
Without it, clang shows a warning during compilation:
`warning: 15 enumeration values not handled in switch: 'TextEntered', 'KeyReleased', 'MouseWheelScrolled'... [-Wswitch]`